samples/deploy/crd/v1beta1/ApisixRoute.yaml

```diff
@@ -44,7 +44,11 @@ spec:
     - name: v1
       served: true
       storage: false
+      deprecated: true
     - name: v2alpha1
+      served: true
+      storage: false
```

Comment:
We may still keep the `v2alpha1` as the storage version, once the v2beta1 logic are completed, we can migrate from v2alpha1 to v2beta1.

Reply:
OK